src/test_l1_hdmi_cec_driver.c

```diff
@@ -600,7 +504,7 @@ void test_hdmicec_hal_l1_getPhysicalAddress_positive( void )
  */
 void test_hdmicec_hal_l1_addLogicalAddress_negative( void )
 {
-    int result;
+    HDMI_CEC_STATUS result = HDMI_CEC_IO_SUCCESS;;
```

Comment:
For all Add and remove logic address positive or negative functions, we need to check for NOT_SUPPORTED error code on all the source devices

Reply:
Addressed the review